openmetadata-docs/content/openmetadata/glossaries/index.md

```diff
@@ -24,4 +24,39 @@ The terms from the glossary can be used for labeling or tagging as additional me
 <Image
 src={"/images/openmetadata/glossaries/glossaries.gif"}
 alt="Glossary page demo"
-/>  
\ No newline at end of file
+/>
+
+## Glossary Bulk upload
+
+With glossary bulk upload you can create/update multiple glossary terms simultaneously. Below are steps for the same.
+
+- Create glossary in which you want to upload glossary terms `csvEg. Business Glossary`.
+- Click on 3 dots and click on export button (if you have terms already you will get CSV file containing all the terms, if you don't have any terms you will get blank CSV template).
```

Comment:
Plz replace with this:
Click on the 3 dots icon and click on the Export button. If you have glossary terms in your Glossary, the same will be exported as a CSV file. If there are no terms in the Glossary currently, then a blank CSV template will be downloaded.